### PR TITLE
feat: add ssh to destinations table

### PR DESCRIPTION
**Pull request by hoyyeva**
## Summary
- add ssh to destinations table

## 📷 
Screen Shot 2022-12-19 at 11 00 16 AM

https://user-images.githubusercontent.com/63033505/208481803-69257409-c06c-45b2-a7d8-8953efa821aa.mov

### CODE REVIEW

**Review by jmorganca — APPROVED** [review]

File: ui/pages/destinations/index.js [hunk 1]
```diff
@@ -45,7 +46,9 @@ export default function Destinations() {
       </header>
 
       <Table
-        href={row => `/destinations/${row.original.id}`}
+        href={row =>
+          row.original.kind === 'ssh' ? '' : `/destinations/${row.original.id}`
```
Comment (jmorganca) [inline ui/pages/destinations/index.js:50]:
```suggestion
          row.original.kind === 'kubernetes' && `/destinations/${row.original.id}`
```

File: ui/pages/destinations/index.js [hunk 2]
```diff
@@ -63,26 +67,32 @@ export default function Destinations() {
...
-                    {info.row.original.resources?.length > 0 && (
-                      <span>
-                        {info.row.original.resources?.length}&nbsp;
-                        {info.row.original.resources?.length === 1
-                          ? 'namespace'
-                          : 'namespaces'}
-                      </span>
-                    )}
-                  </div>
+                  {info.row.original.kind !== 'ssh' && (
```
Comment (jmorganca) [inline ui/pages/destinations/index.js:84]: For `ssh` let's make the secondary text the url (e.g. `row.original.connection.url`) if it exists. That way users know what hostname to ssh into

File: ui/pages/destinations/index.js [hunk 3]
```diff
@@ -23,6 +24,31 @@ export default function Destinations() {
     return null
   }
 
+  const test = [
```
Comment (jmorganca) [inline ui/pages/destinations/index.js:27]: Don't forget to remove me!

Comment (hoyyeva) [reply ui/pages/destinations/index.js:27]: oops! good catch >.<

**Review by jmorganca — APPROVED** [review]

**Pull request merged**